internal/statestore/redis_indices.go

```diff
@@ -56,6 +58,8 @@ func extractIndexedFields(cfg config.View, t *pb.Ticket) indexedFields {
 		}
 	}
 
+	result[allTickets] = float64(time.Now().UnixNano())
```

Comment:
This will have a side-effect (or may be a feature?) that "allTickets" is now a queriable index that can return tickets in a time range :) So some could specify that one filter "allTickets" for range t1 to t2? Should we document it? (not in this PR - but in general documentation for 0.7)

Reply:
Also, can this then just be renamed to "createTime" meta index? always stores time created - and can query based on "createTime" filter too?

Reply:
I just changed it to always be 0, because unit testing with time.Now() doesn't work, and I didn't want to do more plumbing to have real vs stubbed values in the function.

Also, you won't be able to query the allTickets index directly.  That is the point of the index prefixes: It avoids such side-effect behavior.

Reply:
Also, index time doesn't equal create time.  Especially if we make changes to properly support index values changing, it is possible for tickets to get reindexed.

Reply:
Ok. I think as a filter on a ticket conceptually - "alltickets" with value 0 - seems more a workaround for UTs - than "createTime" with value time.Now() at creation time. Also, this could be a cheap way to leverage when we get to the meta index feature that we can expose. Wont block that on this PR - we can always modify it when we get to it.